src/lib/i18n/locales/vi-VN/translation.json

```diff
@@ -0,0 +1,363 @@
+    "Oops! You're using an unsupported method (frontend only). Please serve the WebUI from the backend.": "Rất tiếc! Bạn đang sử dụng một phương thức không được hỗ trợ (chỉ dành cho frontend). Vui lòng cung cấp phương thức cho WebUI từ phía backend.",
+    "Open": "Mở",
+    "Open AI": "Open AI",
+    "Open AI (Dall-E)": "Open AI (Dall-E)",
+    "Open new chat": "Mở cuộc trò chuyện mới",
+    "OpenAI API": "API OpenAI",
+    "OpenAI API Key": "Khóa API OpenAI",
+    "OpenAI API Key is required.": "Yêu cầu Khóa API OpenAI.",
+    "or": "hoặc",
+    "Parameters": "Tham số",
+    "Password": "Mật khẩu",
+    "PDF Extract Images (OCR)": "Trích xuất ảnh từ PDF (OCR)",
+    "pending": "đang chờ",
+    "Permission denied when accessing microphone: {{error}}": "Quyền truy cập micrô bị từ chối: {{error}}",
+    "Playground": "Thử nghiệm (Playground)",
+    "Profile": "Hồ sơ",
+    "Prompt Content": "Nội dung prompt",
+    "Prompt suggestions": "Đề nghị prompt",
+    "Prompts": "prompt",
+    "Pull a model from Ollama.com": "Tải một mô hình từ Ollama.com",
+    "Pull Progress": "Tiến trình Tải xuống",
+    "Query Params": "Tham số Truy vấn",
+    "RAG Template": "Mẫu prompt cho RAG",
+    "Raw Format": "Raw Format",
+    "Record voice": "Ghi âm",
+    "Redirecting you to OpenWebUI Community": "Đang chuyển hướng bạn đến Cộng đồng OpenWebUI",
+    "Release Notes": "Ghi chú Phát hành",
+    "Repeat Last N": "Repeat Last N",
+    "Repeat Penalty": "Repeat Penalty",
+    "Request Mode": "Request Mode",
+    "Reset Vector Storage": "Cài đặt lại Vector Storage",
+    "Response AutoCopy to Clipboard": "Tự động Sao chép Phản hồi vào clipboard",
+    "Role": "Vai trò",
+    "Rosé Pine": "Rosé Pine",
+    "Rosé Pine Dawn": "Rosé Pine Dawn",
+    "Save": "Lưu",
+    "Save & Create": "Lưu & Tạo",
+    "Save & Submit": "Lưu & Gửi",
+    "Save & Update": "Lưu & Cập nhật",
+    "Saving chat logs directly to your browser's storage is no longer supported. Please take a moment to download and delete your chat logs by clicking the button below. Don't worry, you can easily re-import your chat logs to the backend through": "Không còn hỗ trợ lưu trữ nhật ký trò chuyện trực tiếp vào bộ nhớ trình duyệt của bạn. Vui lòng dành thời gian để tải xuống và xóa nhật ký trò chuyện của bạn bằng cách nhấp vào nút bên dưới. Đừng lo lắng, bạn có thể dễ dàng nhập lại nhật ký trò chuyện của mình vào backend thông qua",
+    "Scan": "Quét",
+    "Scan complete!": "Quét hoàn tất!",
+    "Scan for documents from {{path}}": "Quét tài liệu từ {{path}}",
+    "Search": "Tìm kiếm",
+    "Search Documents": "Tìm Tài liệu",
+    "Search Prompts": "Tìm prompt",
+    "See readme.md for instructions": "Xem readme.md để biết hướng dẫn",
+    "See what's new": "Xem những gì mới",
+    "Seed": "Hạt giống",
+    "Select a mode": "Chọn một chế độ",
+    "Select a model": "Chọn một mô hình",
+    "Select an Ollama instance": "Chọn một thực thể Ollama",
+    "Send a Messsage": "Gửi yêu cầu",
```

Comment:
`messsage` was fixed in another PR, you'll need to adjust this to `message` here as well.